src/OpenLoco/Windows/TownList.cpp

```diff
@@ -595,25 +595,25 @@ namespace OpenLoco::Ui::Windows::TownList
 
         static WindowEventList events;
 
+        constexpr string_id townSizeNames[9] = {
+            StringIds::tooltip_select_town_size,
```

Comment:
Why is this string id part of this array?

Reply:
I'm not entirely sure but think it's just the default value before selecting a size

Reply:
I think it defaults to three. Besides, it's a tooltip, right?

Reply:
Hmm no idea then.

Reply:
okay ive removed it.